nexus/src/external_api/console_api.rs

```diff
@@ -809,7 +805,8 @@ pub async fn serve_console_index(
     Ok(Response::builder()
         .status(StatusCode::OK)
         .header(http::header::CONTENT_TYPE, "text/html; charset=UTF-8")
-        .header(http::header::CACHE_CONTROL, cache_control_value(apictx))
+        // do not cache this response in browser
+        .header(http::header::CACHE_CONTROL, "no-store")
         .body(file_contents.into())?)
 }
```

Comment:
Eventually it would be nice to have etag support for this at least so we could 304 if it hasn't changed.